static/app/views/settings/account/accountEmails.tsx

```diff
@@ -33,6 +33,10 @@ class AccountEmails extends AsyncView<Props, State> {
     return t('Emails');
   }
 
+  getEndpoints() {
+    return [];
+  }
```

Comment:
Does this need to be an `AsyncView` if it doesn't have endpoints? -- I supposed we'd have to refactor the page title.

Reply:
Yeah it probably doesn't need to be an AsyncView but that felt like a more invasive change than this.